src/main/g8/src/main/scala/$package__packaged$/Server.scala

```diff
@@ -1,15 +1,18 @@
 package $package$
 
-import fs2.{Task, Stream}
-import org.http4s.util.StreamApp
-import org.http4s.server.Server
+import fs2.Task
+
```

Comment:
Nit - Remove extra line here and double lines after imports

Reply:
extra line vanquished